lua/refactoring/tests/refactor_spec.lua

```diff
@@ -151,6 +193,12 @@ describe("Refactoring", function()
         end)
     end)
 
+    for_each_lsp_file(function(file)
+        it(string.format("Refactoring: %s", file), function()
+            inline_test(file)
```

Comment:
Perhaps inline_test can be renamed to lsp_test

Reply:
Removed function the function and merged into existing one.